src/smoke.hh

```diff
@@ -0,0 +1,19 @@
+#include <nanogui/nanogui.h>
+
+#include "emittor.hh"
+
+using namespace nanogui;
```

Comment:
Avoid `using namespace` in header files. It might pollute namespace of the entire project. (Might not be a big deal in a small project like this one, but still good practice)